main.py

```diff
@@ -1,10 +1,72 @@
 import init_django_orm  # noqa: F401
+import json
 
 from db.models import Race, Skill, Player, Guild
 
 
+def create_or_update_race(race_data: dict) -> Race:
+    race_description = race_data.get("description")
+    if race_description is None:
+        return (Race.objects.get_or_create(name=race_data["name"]))[0]
```

Comment:
What are you doing here? Why are you sure that the name is present if race_description is not?
Just return None if not race_data

Reply:
but according to the conditions of the task: race description -  can be blank. Maybe I'm wrong, but I concluded that race can be without a description. And there can be no players without race. So missing a race_data (which includes the race name and description), will cause an error one way or another